frontend/src/components/question-creation/question-elements/modals/multiple-choice/MultipleChoiceOption.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from "react";
+import {
+  Input,
+  Checkbox,
+  Spacer,
+  Box,
+  Button,
+  IconButton,
+  VStack,
+  HStack,
+  Text,
+} from "@chakra-ui/react";
+import { DeleteOutlineIcon } from "../../../../../assets/icons";
+import { MultipleChoiceOptionData } from "../../../../../types/QuestionTypes";
+import { updatedMultipleChoiceOption } from "../../../../../utils/QuestionUtils";
+
+interface MultipleChoiceOptionProps {
+  data: MultipleChoiceOptionData;
+  isEmptyError: boolean;
+  isCorrectError: boolean;
+  setOptions: React.Dispatch<React.SetStateAction<MultipleChoiceOptionData[]>>;
+  setOptionCount: React.Dispatch<React.SetStateAction<number>>;
+}
+
+const MultipleChoiceOption = ({
+  data,
+  isEmptyError,
+  isCorrectError,
+  setOptions,
+  setOptionCount,
+}: MultipleChoiceOptionProps): React.ReactElement => {
+  const { id, value, isCorrect } = data;
+  const [lengthError, setLengthError] = useState(false);
+
+  const isOversized = (input: string) => input.length > 800;
```

Comment:
```suggestion
  const exceedsMaxLen = (input: string) => input.length > 800;
```

Reply:
if we're extracting this into a function, do we want to put it into utils and refactor the other elements to use it? otherwise i think we can directly embed this logic?

Reply:
put into utils & updated other components to use it! i think its a bit more readable this way